### PR TITLE
SOC-5744:Exceptions of spaces acitivities migration preventing migrat…

### PR DESCRIPTION
**Pull request by bourasom**
…ion of proceeding succesfully

### CODE REVIEW

File: component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java [hunk 1]
```diff
@@ -325,7 +326,12 @@ private void migrationByIdentity(String userName, IdentityEntity identityEntity)
         //owner.setProviderId(providerId);
         //
         activity.setId(null);
-        activity.setTitle(formatHTML(activity.getTitle()));
+        String title = formatHTML(activity.getTitle());
+        if ( title == null || title.equals("") )
+        {
```
Comment (thomasdelhomenie) [inline component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java]: Bracket must be at the end of the line.

File: component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java [hunk 2]
```diff
@@ -66,6 +66,7 @@
   public static final String EVENT_LISTENER_KEY = "SOC_ACTIVITY_MIGRATION";
   private static final Pattern MENTION_PATTERN = Pattern.compile("@([^\\s]+)|@([^\\s]+)$");
   public final static String COMMENT_PREFIX = "comment";
+  public final static String DEFAULT_TITLE = "\t";
```
Comment (boubaker) [inline component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java]: How is this character is interpreted in HTML ? Is it interpreted as a tabulation ?

Comment (bourasom) [inline component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java]: The shown activity is blank

File: component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java [hunk 3]
```diff
@@ -371,10 +371,12 @@ private void migrationByIdentity(String userName, IdentityEntity identityEntity)
...
               comment.setPosterId(getNewIdentityId(comment.getPosterId()));
 
-              if (comment.getTitle() == null) {
-                comment.setTitle("");
-              }else {
-                comment.setTitle(formatHTML(comment.getTitle()));
+              String commentTitle = formatHTML(comment.getTitle());
+              if (commentTitle == null || commentTitle.equals("") ) {
+                comment.setTitle(DEFAULT_TITLE);
+              }
```
Comment (ahamdi) [inline component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/updater/ActivityMigrationService.java]: else should be in the same line with the closing bracket

**Pull request merged**